broker/src/main/java/org/apache/rocketmq/broker/failover/EscapeBridge.java

```diff
@@ -63,44 +96,52 @@ public void start() throws Exception {
         if (brokerController.getBrokerConfig().isEnableSlaveActingMaster() && brokerController.getBrokerConfig().isEnableRemoteEscape()) {
             String nameserver = this.brokerController.getNameServerList();
             if (nameserver != null && !nameserver.isEmpty()) {
```

Comment:
The nameserver condition no longer seems necessary

Reply:
Exactly~ I've removed it.